README.md

```diff
@@ -1,77 +1,80 @@
+3. Fetch and install submodule dependencies
+```shell
+git submodule init
+git submodule update --recursive
+```
+4. Rust the a-s Rust unit tests
+```shell
+cargo test
+```
 
-If that's not why you're here, then instead you might enjoy:
+### Consumer build, integration and testing
+The application-services library primary consumers are Fenix (Firefox on Android) and Firefox iOS. Assure you are able to run integration tests (for Android and iOS if using MacOS) by following the instructions to build for Android and iOS integrations.  
 
-* The [Application Services Product
-  Portal](https://mozilla.github.io/application-services/), if you're looking to
-  use those products in your application.
-* The [Application Services Team Home](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home)
-  on Mana, if you're trying to find out more about how we build them.
+#### Android integration builds and helpful tools
+* Build instructions to test [Fenix / android-components integration](https://github.com/mozilla/application-services/blob/main/docs/build.md#building-for-fenix) (2 hrs)
+* [Fenix Auto-publication workflow for android-components and application-services](https://github.com/mozilla-mobile/fenix/#auto-publication-workflow-for-android-components-and-application-services)
+* [Nimbus: using local servers for development](https://github.com/mozilla-mobile/fenix/#using-nimbus-servers-during-local-development)
 
 
-### Overview
+#### Firefox for iOS integration
+* Build instructions to test [Firefox iOS integration](https://github.com/mozilla/application-services/blob/main/docs/build.md#building-for-firefox-ios) (2 hrs)
 
-This repository is used to build client-side libraries for integrating with
-Firefox Application services such as Firefox Accounts, Firefox Sync and Push.
-Each of these is called a "component" and is built using a core of shared code
-written in Rust, wrapped with native language bindings for different platforms.
+#### Firefox Desktop
+* Build instructions to test [Firefox Desktop integration](https://github.com/mozilla/application-services/blob/main/docs/build.md#building-for-firefox-desktop) (?? hrs)
 
-The end result is an application that can be assembled from re-usable components
-that are largely shared across platforms, like this:
+### Documentation
+We use rustdoc to document both the public API of the component and the various internal implementation details. Once you have completed the build steps, you can view the docs by running:
 
-[![component diagram](https://docs.google.com/drawings/d/e/2PACX-1vTPOIIBsqvkWfecYOziEnv-hrkB9QbpZwcHyeyUB-p3-eP1w9L87vwnJMiGt-eO5r-K-XcHPl_YwjvU/pub?w=727&h=546)](https://docs.google.com/drawings/d/1WRv2AaOsutNdL8_E5UDsYg1sC6FKRJ9P0bBSoI7E19s/)
+```shell
+cargo doc --no-deps --document-private-items --open
```

Comment:
I'm not sure how well this will work in the root of a workspace, but it'll be interesting to play around with!

Reply:
Some discussion of limitations of running in a workspace here: https://github.com/rust-lang/cargo/issues/8229

Reply:
Would it make sense to pull what Vlad did in a-s-dev (https://a-s-dev.github.io/as-docs/fxa_client/index.html) to host the docs on a gh page into the a-s repository?